streamwebs_frontend/streamwebs/templates/streamwebs/sites.html

```diff
@@ -0,0 +1,169 @@
+{% extends "streamwebs/base.html" %}
+
+{% load staticfiles %}
+
+{% block title %}Sites - Streamwebs{% endblock %}
+
+{% block scripts %}
+<script src="http://maps.google.com/maps/api/js"></script>
+<script type="application/javascript">
```

Comment:
For a small, loose collection of page-specific functions. I can move it out into a separate file, but given that the file would pretty much only be used by this page, I'm not sure I see the point.